examples/unitTestExample/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.realm.examples.intro" >
```

Comment:
package name should be different to avoid conflicts with the intro example. Perhaps `io.realm.examples.testing`. `test` is used when running instrumentation tests, so using that is probably a bad idea?